source/extensions/filters/http/common/stream_rate_limiter.h

```diff
@@ -0,0 +1,93 @@
+#pragma once
```

Comment:
This refactor should probably be in a separate PR as well.

Reply:
I was following the change [#13395](https://github.com/envoyproxy/envoy/commit/5d12b675fe13747bc716859275c2d3c21004e99f#diff-1193859f9acae8da94103e4e9210089e429439f5acbd0e6958faeae731a7a9d9) that also did similar refactoring in the same change. I understand the comment for token bucket changes but for this change it should be ok to be reverted if the filter change is reverted.